portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,15 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+/**
+ * The above code is organized to show each individual step, but we can use an
+ * ES6 feature called arrow functions to shorten the code. This function
+ * combines all of the above code into a single Promise chain. You can use
+ * whichever syntax makes the most sense to you.
+ */
+function getRandomQuoteUsingArrowFunctions() {
```

Comment:
I would update the comment to just say what it does. So you can say " Fetches a random quote from the server." If you want a reminder to update this function to use lambdas I would just add a todo to the comment, like, TODO(psharma): Update this method to use Lambdas.

I would also rename the function to just getRandomQuotes. Whether you use Lambda/Arrow functions is an implementation detail and it is better not to expose that.